src/components/Shared/LinkToHome.tsx

```diff
@@ -1,19 +1,31 @@
 import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
 import Link from '@mui/material/Link';
 import Stack from '@mui/material/Stack';
+import { Box } from '@mui/system';
+import { style } from 'typestyle';
+
+import { Spacing } from '../../styles';
 
 export function LinkToHome() {
+  const styles = {
+    box: style({
+      marginBottom: `${Spacing.Small + 1}px`,
+    }),
+  };
+
   return (
-    <Link href='/' aria-label='link to home'>
-      <Stack
-        direction='row'
-        sx={{
-          alignItems: 'center',
-        }}
-      >
-        <ChevronLeftIcon fontSize='small' />
-        <p>Home</p>
-      </Stack>
-    </Link>
+    <Box className={`${styles.box}`}>
```

Comment:
This can be `className={styles.box}` No need for the template literals here since we don't have a string name for the className